.circleci/config.yml

```diff
@@ -246,7 +246,7 @@ build_and_deploy: &build_and_deploy
           terminus build:workflow:wait $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV "Sync code on $PANTHEON_ENV"
 
           echo "Rebuilding cache..."
-          terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr
+          terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr || true
```

Comment:
Just curious: is the rebuilding cache command failing from time to time?

Reply:
After this module is updated, `drush cr` fails if it's run before the database updates are performed. 
![image](https://github.com/user-attachments/assets/f3c174cf-bc59-4d8e-a9ee-48128b286993)